api/services/GithubBuildStatusReporter.js

```diff
@@ -107,7 +107,7 @@ const reportBuildStatus = (build) => {
       context,
     };
 
-    if (build.state === 'processing') {
+    if (['processing', 'queued'].find(s => s === build.state)) {
```

Comment:
Could this use `includes` instead? Ex:
```
 if (['processing', 'queued'].includes(build.state))
```

Reply:
good suggestion ... changed ... thanks 👍